cajita/unit_test/tstSparseIndexSpace.hpp

```diff
@@ -0,0 +1,386 @@
+#include <Cajita_SparseIndexSpace.hpp>
+
+#include <Kokkos_Core.hpp>
+
+#include <gtest/gtest.h>
+
+using namespace Cajita;
+
+namespace Test
+{
+//---------------------------------------------------------------------------//
+void testAtomicOr()
+{
+    constexpr int size = 64;
+
+    Kokkos::View<bool ** * [2], TEST_MEMSPACE> _tile_insert_record(
```

Comment:
Let's try changing the `bool` to `int` here to see if that is the issue.

Reply:
Per @dalg24 we suspect we might be hitting this: https://github.com/kokkos/kokkos/blob/35613e69dcda1dd2274b66fa282d623add732dea/core/src/impl/Kokkos_Atomic_Generic.hpp#L403

Possibly the string showing the error message wasn't showing up in our log.

Reply:
Please refer to this commit [cc716ad](https://github.com/ECP-copa/Cabana/pull/287/commits/cc716ad1eaa228171defdbbe8544f8fd89a53331)

Reply:
Works when changing `bool` to `int`